Userland/Libraries/LibJS/Runtime/AtomicsObject.cpp

```diff
@@ -440,6 +441,29 @@ JS_DEFINE_NATIVE_FUNCTION(AtomicsObject::or_)
     VERIFY_NOT_REACHED();
 }
 
+// Atomics.pause ( [ N ] )
```

Comment:
Please include the spec link in the same format as the other prototypes

Reply:
This function is not in the spec yet. I will add the link to the proposal, http://tc39.es/proposal-atomics-microwait/.

Reply:
Yep, that's what we do everywhere else (including spec numbers, even if it's just `1` in this case). e.g. see:
https://github.com/LadybirdBrowser/ladybird/blob/d634039c1010b54fbdf76a6768fea2ae6986b5ec/Userland/Libraries/LibJS/Runtime/Intl/DurationFormatPrototype.cpp#L39-L40